exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+var htmlLevelAbove5 = htmlLevel > 5 ;
+var cssLevelAbove5 =cssLevel > 5;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+var cssAndHtmlAbove5 = htmlLevel > 5 && cssLevel > 5;
+var cssOrHtmlAbove5 = htmlLevel  > 5 && htmlLevel > 5;
 
```

Comment:
Just a little heads up here Bimbola, in the comments it says 'Do not hardcode'. as you have already assigned the 2 variables htmlLevelAbove5 and cssLevelAbove5, you can use those variables to declare the last 2 variables. I did the same thing and then realised if the value of htmlLevel or cssLevel changed then I would have to change the rest of the code as well.